### ISSUE TITLE
swank client attempt to call a string value

### ISSUE THREAD

**Issue by glyh** [Olical/conjure#311]
Here's the error message:
```
Error detected while processing ExitPre Autocommands for "*":                                                                  
E5108: Error executing lua ...ite/pack/packer/opt/conjure/lua/conjure/aniseed/core.lua:137: ...ker/opt/conjure/lua/conjure/clie
nt/common-lisp/swank.lua:40: attempt to call a string value                                                                    
stack traceback:                                                                                                               
        [C]: in function 'f'                                                                                                   
        ...ite/pack/packer/opt/conjure/lua/conjure/aniseed/core.lua:137: in function 'on-exit'                                 
        [string ":lua"]:1: in main chunk  
```

To reproduce:
1. Do not open any remote server
2. Open https://github.com/joaotavora/sly/blob/master/slynk/slynk.lisp
3. `:wq` to save + quit.

**Comment by Dotrar** [comment]
This seems to be that I just didn't define the exit callback correctly; I'm a little too swamped to try and work on it now but I'll see if I can fix it up shortly?

**Comment by Dotrar** [comment]
nvim it was simple error: I fixed it in #313

**Comment by Olical** [comment]
Merged the fix, will release today!
